health/health.d/am2320.conf

```diff
@@ -1,12 +1,15 @@
 # make sure am2320 is sending stats
 
-template: am2320_last_collected_secs
-      on: am2320.temperature
-    calc: $now - $last_collected_t
-   units: seconds ago
-   every: 10s
-    warn: $this > (($status >= $WARNING)  ? ($update_every) : ( 5 * $update_every))
-    crit: $this > (($status == $CRITICAL) ? ($update_every) : (60 * $update_every))
-   delay: down 5m multiplier 1.5 max 1h
-    info: number of seconds since the last successful data collection
-      to: webmaster
\ No newline at end of file
+ template: am2320_last_collected_secs
+       on: am2320.temperature
+    class: Other
+component: Sensors
+     type: Latency
+     calc: $now - $last_collected_t
+    units: seconds ago
+    every: 10s
+     warn: $this > (($status >= $WARNING)  ? ($update_every) : ( 5 * $update_every))
+     crit: $this > (($status == $CRITICAL) ? ($update_every) : (60 * $update_every))
+    delay: down 5m multiplier 1.5 max 1h
+     info: number of seconds since the last successful data collection
+       to: webmaster
```

Comment:
Please, add a new line at the end of file.

Reply:
Done! (https://github.com/netdata/netdata/pull/10961/commits/51b8857c35498b14bed043b672aa79683a7c76ed). Thanks!